tests/falco/commands_test.go

```diff
@@ -132,3 +134,38 @@ func TestFalco_Cmd_PluginInfo(t *testing.T) {
 			`No suggested open params available.*`),
 		res.Stdout())
 }
+
+func TestFalco_Print_IgnoredEvents(t *testing.T) {
+	t.Parallel()
+	checkDefaultConfig(t)
+	type Data struct {
+		Events []string `json:"events"`
+	}
+	evntfile, err := os.Open("events.json")
+	if err != nil {
+		panic(err)
+	}
+	defer evntfile.Close()
+
+	var events Data
+	err = json.NewDecoder(evntfile).Decode(&events)
+	if err != nil {
+		panic(err)
+	}
+
+	runner := tests.NewFalcoExecutableRunner(t)
+	t.Run("all-events-ignored-by-default", func(t *testing.T) {
+		res := falco.Test(
+			runner,
+			falco.WithArgs("-i"),
+		)
+		assert.Regexp(t, regexp.MustCompile(
+			`Ignored\sEvent\(s\):\n`,
+		), res.Stdout())
+		for _, event := range events.Events {
+			assert.Regexp(t, regexp.MustCompile(`\b`+event+`\b`), res.Stdout())
```

Comment:
Maybe `assert.Contains` would be more lightweight here, since events are simple known strings. Remember, the more we make tests optimized, the less the CI will take to run them!